renderMath.js

```diff
@@ -78,13 +87,12 @@ function renderMath(parentElement = "") {
             <script id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
 
             */
-            var polyfillScript = document.createElement("script");
-            polyfillScript.setAttribute("src", "https://polyfill.io/v3/polyfill.min.js?features=es6");
+            // var polyfillScript = document.createElement("script");
+            // polyfillScript.setAttribute("src", "https://polyfill.io/v3/polyfill.min.js?features=es6");
             var mathjaxScript = document.createElement("script");
-            mathjaxScript.setAttribute("id", "MathJax-script");
+            mathjaxScript.setAttribute("type", "text/javascript");
             mathjaxScript.setAttribute("async", "");
-            mathjaxScript.setAttribute("src", "https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js");
-            document.head.appendChild(polyfillScript);
```

Comment:
If there is no polyfill script, either comment this line out like you did with the others, or delete the other two commented lines out above (regarding the polyfill script), we've gotta be consistent in the changes we make